### DISCUSSION TITLE
关于源码与文档不对应问题

### DISCUSSION THREAD

**Discussion by mistery-li** [element-plus/element-plus#2049] [Q&A]
目前正在计划为UI库补充相关测试用例。但是发现部分问题
    在utils工具目录下面发现写了一个toObject函数，只有scrollbar会用到，但是https://element-plus.gitee.io/#/zh-CN/component/scrollbar文档下面的Scrollbar Attributes里面wrap-style文档是只能传递string参数。源码里面发现props是支持传递Array的，测试发现会报错。目前应该以代码为准，还是文档？

另：刚开始学习参与开源，不是很了解各种操作流程，以及需要和团队沟通，讨论有什么渠道？以及哪里可以了解团队的计划？

**Comment by iamkun** [comment]
feel free to open an issue if you have any questions.

And thanks for contributing to Element Plus.

**Comment by mistery-li** [comment]
[image: image.png]
本来是打算给toObject写一下测试的。关于滚动条这里，我看到文档是只支持参数string，但是源码实现这里是支持了Array的，所以不是很了解这里的用法。就不太了解需要怎么改进[image:
image.png]

iamkun ***@***.***> 于2021年5月24日周一 上午11:13写道：

**Reply by iamkun** [reply]
You can update the docs if needed. 😁

**Comment by jw-foss** [comment]
Simply open issue to us so that we can see what was bothering you, if you are interested in contributing to the project. Open a PR and add @element-plus/backers as the reviewers, we will get your PR reviewed in time(or maybe a little bit late depends on how much time the team has).
